scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -273,8 +273,18 @@ class FossIdTest : WordSpec({
             val summary = fossId.scan(listOf(createPackage(pkgId, vcsInfo))).summary(pkgId)
 
             val expectedIssues = listOf(
-                OrtIssue(timestamp = Instant.EPOCH, createPendingFile(4), "pending", Severity.HINT),
-                OrtIssue(timestamp = Instant.EPOCH, createPendingFile(5), "pending", Severity.HINT)
+                OrtIssue(
+                    timestamp = Instant.EPOCH,
+                    createPendingFile(4),
```

Comment:
How about calling `createPendingFile(X)` for each `X` only once, like

    val expectedIssues = listOf(createPendingFile(4), createPendingFile(5)).map {
        OrtIssue(Instant.EPOCH, $it, "Pending identification for '$it'.", Severity.HINT)
    }